RacingAidWpf/View/MainWindow.xaml

```diff
@@ -38,11 +38,22 @@
                     Command="{Binding StopCommand}"
                     IsEnabled="{Binding IsStarted}"/>
             <ToggleButton x:Name="RepositionButton"
-                          Height="25" Width="120"
-                          Margin="10, 0, 0, 0"
-                          Content="Enable Reposition"
-                          Command="{Binding MoveOverlaysToggleCommand}"
-                          IsEnabled="{Binding IsStarted}"/>
+              Height="25" Width="120"
+              Margin="10, 0, 0, 0"
+              IsChecked="{Binding IsRepositionEnabled}" 
+              IsEnabled="{Binding IsStarted}">
+            <ToggleButton.Style>
+                <Style TargetType="ToggleButton">
+                    <Setter Property="Content" Value="Enable Reposition" />
```

Comment:
Does this not need to be in a similar DataTrigger like below but for `False` values rather than `True`?